### DISCUSSION TITLE
添加自定义的draft scaffold至`_drafts`

### DISCUSSION THREAD

**Discussion by realzza** [ppoffice/hexo-theme-icarus#1174] [Q&A]
如题，希望设置不同的draft模版，通过`hexo new draft-template-name "John doe"`使用。其中`draft-template-name`为自定义的draft模版。希望可以将不同draft模版映射到`_drafts`目录中

**Comment by scutLMG** [comment]
没看懂"将不同draft模版映射到_drafts目录"，不是直接在scafflods文件夹里创建模板直接用吗？

**Reply by realzza** [reply]
比如你在scaffolds里自定义了`self-defined-draft.md`, 执行`hexo new self-defined-draft "test draft template"`，得到的文件会在`_posts`目录下，而不会存在于`_drafts`目录下。换言之目前仅支持给post加模版，不支持给draft加模版。

**Reply by scutLMG** [reply]
原来是这个意思，你是想弄个草稿，暂不发表。你的需求应该去hexo提问。
不过我看了下似乎是有解决办法的：https://hexo.io/docs/writing.html#Layout
“There are three default layouts in Hexo: post, page and draft. Files created by each of them is saved to a different path. Newly created posts are saved to the source/_posts folder.”
所以大概你只能修改scaffolds里的draft.md来实现，写草稿的时候 hexo new draft “artical-name” 来实现。
